crates/subspace-erasure-coding/src/lib.rs

```diff
@@ -59,7 +59,7 @@ impl ErasureCoding {
     pub fn recover(&self, shards: &[Option<Scalar>]) -> Result<Vec<Scalar>, String> {
         // TODO This is only necessary because upstream silently doesn't recover anything:
         //  https://github.com/sifraitech/rust-kzg/issues/195
-        if shards.iter().filter(|scalar| scalar.is_some()).count() < shards.len() / 2 {
+        if shards.iter().filter(|scalar| scalar.is_none()).count() > shards.len() / 2 {
```

Comment:
I think this change deserves some comment for readability purposes since this change threw me off completely why the comparision changed until i looked at the filter again

Reply:
Issue mentioned above actually got resolved early today by https://github.com/sifraitech/rust-kzg/pull/212, I'll simply remove this workaround soon, it was just not handled by underlying library before.